### ISSUE TITLE
In PowerPoint, NVDA cannot give information about rotated placeholders

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#10434]
### Is your feature request related to a problem? Please describe.

If a placeholder covers (or is behind) another placeholder, NVDA generally reports this when navigating to, or moving the placeholder.  NVDA also reports the new position when moving placeholders (eg with the arrow keys).

NVDA does not report anything when rotating placeholders with alt+left and right arrows.
NVDA does not report when a rotated placeholder is behind or covering another placeholder.  The report of this information is based on where the placeholder would be at 0 degrees rotation.

### Describe the solution you'd like

NVDA should report the value of "rotation" (Presented in the format shape pane) when this changes (with alt+left and right arrows).

NVDA should report when rotated placeholders actually obscure other placeholders.

### Additional context
Narrator on Windows 10 does not report this information either, so it may well be that PowerPoint is not exposing the information for NVDA to report
